### DISCUSSION TITLE
Tailwind CSS doesnt work, i use app router with i18n(bilingual) in next.js

### DISCUSSION THREAD

**Discussion by marcoAu109** [tailwindlabs/tailwindcss#12367] [Help] [answered]
Hi Please help!

When i set up bilingual i18next under App/[lng] , once i move the page.js under the [lng] directory ,the tailwind CSS doesnt work.

i understand we may need to set module.exports content in tailwindcss.config. does tailwind css wont support under i18next [lng] situation?

but i cant figure it out 

my app router structure

<img width="341" alt="Screenshot 2023-11-06 at 9 50 29 AM" src="https://github.com/tailwindlabs/tailwindcss/assets/113471149/48db557b-a6cd-4db5-b46e-a07672c46d15">

my tailwindcss.config

<img width="748" alt="Screenshot 2023-11-06 at 9 54 14 AM" src="https://github.com/tailwindlabs/tailwindcss/assets/113471149/4d54d47b-349e-4433-b9a5-670b7f3b4d3a">

my page,js pretty simple

<img width="418" alt="Screenshot 2023-11-06 at 9 55 05 AM" src="https://github.com/tailwindlabs/tailwindcss/assets/113471149/20ccffcc-b52f-4dde-8d9e-520c9bfebc15">

all the above works on files under App router , but after adding sub directory [lng] which is the set up of i18next using in app router,

the tailwindCSS cannot be used under [lng],  my [lng] is 'en' and 'zh-Hant'

Anyone, please help. thank you !

**Comment by marcoAu109** [comment, marked as answer]
Silly me.  It is due to import ../globals.css in the page.js under [lng]

cos the path changed, with globals.css cannot inject tailwinds , and after fix the path, Tailwind CSS works well even under[lng].

Thanks @wongjn anyway

and still put the post here if someone may be as careless as me!

**Reply by arnotixe** [reply]
or helps someone that wrote app router from scratch and forgot to import globals.css into the root layout… Thanks! :+1:

**Comment by busma-tech** [comment]
I was looking for answer for my problem, and I got here a hint. In my case, I followed the instructions from this youtube video https://youtu.be/7C3l4wIq8vQ?si=uUQw8gJPg9pw_r1H and created a src folder wrapping the app. My brain hurt till I thought about the tailwind.config.ts file. Once I did the modification there, it worked perfectly.
